From 5eb4509a6651d19c7a28c4506d6aa582c9ee095a Mon Sep 17 00:00:00 2001
From: Jakub Jelinek <jakub@redhat.com>
Date: Wed, 1 Feb 2006 09:30:43 +0000
Subject: [PATCH] 128-bit long double fixes

* include/bits/stdlib-ldbl.h: New file.
* include/bits/wchar-ldbl.h: New file.

---
 ChangeLog                  |    5 +++++
 include/bits/stdlib-ldbl.h |    1 +
 include/bits/wchar-ldbl.h  |    1 +
 3 files changed, 7 insertions(+), 0 deletions(-)
 create mode 100644 include/bits/stdlib-ldbl.h
 create mode 100644 include/bits/wchar-ldbl.h

diff --git a/include/bits/stdlib-ldbl.h b/include/bits/stdlib-ldbl.h
new file mode 100644
index 0000000..6250949
--- /dev/null
+++ b/include/bits/stdlib-ldbl.h
@@ -0,0 +1 @@
+#include <stdlib/bits/stdlib-ldbl.h>
diff --git a/include/bits/wchar-ldbl.h b/include/bits/wchar-ldbl.h
new file mode 100644
index 0000000..29baa2f
--- /dev/null
+++ b/include/bits/wchar-ldbl.h
@@ -0,0 +1 @@
+#include <wcsmbs/bits/wchar-ldbl.h>
